### DISCUSSION TITLE
How can I save a gt using gt.save?

### DISCUSSION THREAD

**Discussion by Sat1ro** [posit-dev/great-tables#253] [Q&A] [answered]
Hello everyone! 

I’m quite new to python, so this must be a redundant question, but I’m having trouble saving the table I’m trying to make, specifically with the self argument. This is what I have: 

> gt.save(file=‘BC_crec_4m.png’, selector=‘table’, scale=1.0, expand=5, window_size=(6000, 6000)) 

after running, it says the self argument is required, thought I’m not sure how to incorporate it. Any help?

**Comment by jrycw** [comment, marked as answer]
@Sat1ro, `gt` should be an instance of `GT`. For example:
```python
from great_tables import GT
from great_tables.data import countrypops

gt = GT(countrypops.head())
gt.save("BC_crec_4m.png", selector='table', scale=1.0, expand=5, window_size=(6000, 6000))
```

**Comment by jrycw** [comment]
I'll close the discussion since the issue appears to be resolved.
